### PR TITLE
MOBN-945, MOBN-951 | New date picker component and session redirect

### PR DESCRIPTION
**Pull request by bsneha90**

### CODE REVIEW

File: ui/react-components/components/AddAppointment/AddAppointment.jsx [hunk 1]
```diff
@@ -448,9 +451,12 @@ const AddAppointment = props => {
                                     updateAppointmentDetails({appointmentDate: date});
                                     updateErrorIndicators({appointmentDateError: !date});
                                 }}
-                                onClear={() => updateAppointmentDetails({appointmentDate: undefined})}
-                                defaultValue={appointmentDetails.appointmentDate}
-                                minDate={getYesterday()}/>
+                                onClear={() => {
+                                    console.log('clear clear')
```
Comment (binduak) [inline ui/react-components/components/AddAppointment/AddAppointment.jsx]: @bsneha90 please remove the console log

File: ui/react-components/components/DatePicker/DateInput.jsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    disable
+} from './DateInput.module.scss'
+import classNames from 'classnames';
+import {isValidUSDate} from '../../utils/DateUtil'
+import PropTypes from "prop-types";
+import moment from "moment";
+
+const DateInput = (props) =>{
+    const validateDate= ({dateValue, minDate, maxDate}) =>{
+        const isDateFormatValid = isValidUSDate(dateValue) || dateValue==='';
```
[the remark this answers is not in the document]
Comment (bsneha90) [reply ui/react-components/components/DatePicker/DateInput.jsx]: That is what was the requirement.

File: ui/react-components/components/AddAppointment/AddAppointment.jsx (hunk 1 above)
Comment (bsneha90) [reply ui/react-components/components/AddAppointment/AddAppointment.jsx]: Will remove, thanks

File: ui/react-components/components/DatePicker/DateInput.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (bsneha90) [reply ui/react-components/components/DatePicker/DateInput.jsx]: isValidUSDate, yes 'mm/dd/yyyy' is US date format

**Pull request merged**